src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -364,6 +443,84 @@ private TerminalBasicInfo(TermInfo.Database db)
+                if (db != null)
+                {
+                    AddKey(db, TermInfo.Database.KeyF1 , ConsoleKey.F1);
+                    AddKey(db, TermInfo.Database.KeyF2 , ConsoleKey.F2);
+                    AddKey(db, TermInfo.Database.KeyF3 , ConsoleKey.F3);
+                    AddKey(db, TermInfo.Database.KeyF4 , ConsoleKey.F4);
+                    AddKey(db, TermInfo.Database.KeyF5 , ConsoleKey.F5);
+                    AddKey(db, TermInfo.Database.KeyF6 , ConsoleKey.F6);
+                    AddKey(db, TermInfo.Database.KeyF7 , ConsoleKey.F7);
+                    AddKey(db, TermInfo.Database.KeyF8 , ConsoleKey.F8);
+                    AddKey(db, TermInfo.Database.KeyF9 , ConsoleKey.F9);
+                    AddKey(db, TermInfo.Database.KeyF10 , ConsoleKey.F10);
+                    AddKey(db, TermInfo.Database.KeyF11 , ConsoleKey.F11);
+                    AddKey(db, TermInfo.Database.KeyF12 , ConsoleKey.F12);
+                    AddKey(db, TermInfo.Database.KeyF13 , ConsoleKey.F13);
+                    AddKey(db, TermInfo.Database.KeyF14 , ConsoleKey.F14);
+                    AddKey(db, TermInfo.Database.KeyF15 , ConsoleKey.F15);
+                    AddKey(db, TermInfo.Database.KeyF16 , ConsoleKey.F16);
+                    AddKey(db, TermInfo.Database.KeyF17 , ConsoleKey.F17);
+                    AddKey(db, TermInfo.Database.KeyF18 , ConsoleKey.F18);
+                    AddKey(db, TermInfo.Database.KeyF19 , ConsoleKey.F19);
+                    AddKey(db, TermInfo.Database.KeyF20 , ConsoleKey.F20);
+                    AddKey(db, TermInfo.Database.KeyF21 , ConsoleKey.F21);
+                    AddKey(db, TermInfo.Database.KeyF22 , ConsoleKey.F22);
+                    AddKey(db, TermInfo.Database.KeyF23 , ConsoleKey.F23);
+                    AddKey(db, TermInfo.Database.KeyF24 , ConsoleKey.F24);
+                    AddKey(db, TermInfo.Database.KeyBackspace , ConsoleKey.Backspace);
+                    AddKey(db, TermInfo.Database.KeyClear, ConsoleKey.Clear);
+                    AddKey(db, TermInfo.Database.KeyDown , ConsoleKey.DownArrow);
+                    AddKey(db, TermInfo.Database.KeyHome , ConsoleKey.Home);
+                    AddKey(db, TermInfo.Database.KeyLeft , ConsoleKey.LeftArrow);
+                    AddKey(db, TermInfo.Database.KeyPageDown , ConsoleKey.PageDown);
+                    AddKey(db, TermInfo.Database.KeyPageUp , ConsoleKey.PageUp);
+                    AddKey(db, TermInfo.Database.KeyRight , ConsoleKey.RightArrow);
+                    AddKey(db, TermInfo.Database.KeyEnd , ConsoleKey.End);
+                    AddKey(db, TermInfo.Database.KeyEnter , ConsoleKey.Enter);
+                    AddKey(db, TermInfo.Database.KeyHelp, ConsoleKey.Help);
+                    AddKey(db, TermInfo.Database.KeyPrint, ConsoleKey.Print);
+                    AddKey(db, TermInfo.Database.KeyInsert, ConsoleKey.Insert);
+                    AddKey(db, TermInfo.Database.KeyDelete , ConsoleKey.Delete);
```

Comment:
Nit: we don't need the space in each line between the database name and the console key.